templates/core/stories.html

```diff
@@ -0,0 +1,295 @@
+{% extends "core/base_core.html" %}
+
+{% block title %}Success Stories | Ubuntu Core{% endblock %}
+{% block meta_description %}Case studies and customer stories of products launched with Ubuntu Core, a secure, fully containerised Ubuntu for embedded devices that is optimised for reliable updates.{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="u-fixed-width">
+    <h1>Celebrating smart innovation</h1>
+    <p class="p-heading--4">Our customers transform industries with Ubuntu Core</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Ubuntu Core in industrial applications</h2>
+    <div class="p-strip is-shallow row u-equal-height u-no-padding--left u-no-padding--right">
```

Comment:
Hey, I've noticed some unusual mix of classnames here.

I wouldn't recommend mixing `p-strip` and `row` on same element. While it seems to work, they are not meant to be on same element, and it can break later (if we change things in Vanilla).

The structure should be:

```html
<div class="p-strip is-shallow">
   <div class="row u-equal-height">
   </div>
</div>
```

Also, as you are using rows and cols here, on the top level of this section you should also use grid classes (row) instead of `u-fixed-width`.
`u-fixed-width` is meant only for situation where you have one section using whole width of the grid. Once you start using grid inside, it should be grid altogether. Or you should only keep title inside `u-fixed-width` element, and wrap the rest in individual rows (especially that you use nested strips for additional padding).


So this will make it something like:
```html
<section class="p-strip--light">
   <div class="u-fixed-width">
       <h2>Ubuntu Core in industrial applications</h2>
   </div>
   <div class="p-strip is-shallow">
       <div class="row u-equal-height"> <!-- proper HTML structure should also mean you don't need u-no-padding here -->
          <!-- cols here -->
       </div>
     </div>
  </div>
</section>
```

Also, for the borders between the section you can consider using bordered strips or `p-separator` (which also provides padding consistent with strips, so may allow you to avoid nested strips).

Sorry for such long comment, but hope that helps simplify this markup a bit.

Reply:
Hi, thanks for the review and explanation (very helpful) - I've applied the changes as you suggested 👍🏼

Reply:
thanks @bartaz !